apps/condo/domains/organization/components/OrganizationSelect.tsx

```diff
@@ -33,49 +38,48 @@ function compareEmployees (lhs: OrganizationEmployeeType, rhs: OrganizationEmplo
 
 const DROPDOWN_OVERLAY_STYLES: CSSProperties = { maxWidth: 300, width: '100%' }
 
-const ORGANIZATION_EMPLOYEE_WHERE_QUERY: OrganizationEmployeeWhereInput = {
-    isRejected: false,
-    isBlocked: false,
-    organization: { type_not: OrganizationTypeType.Holding },
-}
-
 export const InlineOrganizationSelect: React.FC = () => {
     const intl = useIntl()
     const ChooseOrganizationMessage = intl.formatMessage({ id: 'pages.organizations.ChooseOrganizationLabel' })
     const AddOrganizationTitle = intl.formatMessage({ id: 'pages.organizations.CreateOrganizationButtonLabel' })
 
+    const { persistor } = useCachePersistor()
     const router = useRouter()
 
     const { breakpoints } = useLayoutContext()
     const textSize: TypographyTextProps['size'] = !breakpoints.TABLET_LARGE ? 'small' : 'medium'
 
     const { user } = useAuth()
-    const { link, selectLink, isLoading: organizationLoading } = useOrganization()
-    const userId = get(user, 'id', null)
-
-    const { objs: userEmployees, allDataLoaded: employeesLoaded } = OrganizationEmployee.useAllObjects({
-        where: {
-            user: { id: userId },
-            isAccepted: true,
-            ...ORGANIZATION_EMPLOYEE_WHERE_QUERY,
-        },
-    }, { skip: !userId })
+    const {
+        employee: activeEmployee,
+        organization,
+        selectEmployee: setActiveEmployee,
+        isLoading: organizationLoading,
+    } = useOrganization()
+    const userId = user?.id || null
+
+    const { data: actualEmployeesData, loading: isActualEmployeeLoading } = useGetActualOrganizationEmployeesQuery({
+        variables: { userId },
+        skip: !userId || !persistor,
+    })
+    const actualEmployees = useMemo(
+        () => Array.isArray(actualEmployeesData?.actualEmployees) ? actualEmployeesData.actualEmployees.filter(nonNull) : []
+        , [actualEmployeesData]
+    )
 
-    const { count: hasInvites, loading: isInvitesLoading } = OrganizationEmployee.useCount({
-        where: {
-            user: { id: userId },
-            isAccepted: false,
-            ...ORGANIZATION_EMPLOYEE_WHERE_QUERY,
-        },
-    }, { skip: !userId })
+    const { data: invites, loading: isInvitesLoading } = useGetEmployeeInvitesCountQuery({
+        variables: { userId },
+        skip: !userId || !persistor || !!activeEmployee?.id || actualEmployees.length > 0,
```

Comment:
why we skip invites query when we have `activeEmployee.id` or `actualEmployees.length > 0` ?

Reply:
because this request is only used to display the modal window with the creation of an organization when the user is not yet a member of any organization